server/middlewares/validateAuth.js

```diff
@@ -48,5 +50,66 @@ class validateAuth {
       });
     }
   }
+
+  /**
+     * This method handle the reset password request.
```

Comment:
Consider updating this comment as well. It validates instead.

Reply:
updated